pkg/logs/input/tailer/file_provider_windows.go

```diff
@@ -0,0 +1,22 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build windows
+
+package tailer
+
+import (
+	"os"
+)
+
+// exists returns true if the file at path filePath exists
+// Behavior is the same as on unix, but we can't rely on os.IsNotExist for windows.
+// As we're tailing with *, the error is related to the path being malformed.
+func (p *FileProvider) exists(filePath string) bool {
```

Comment:
I think we should just keep this implementation, no need to have two distinct ones.